clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/UpsertGoogleLoadBalancerDescription.groovy

```diff
@@ -26,6 +29,8 @@ class UpsertGoogleLoadBalancerDescription extends AbstractGoogleCredentialsDescr
   String portRange
   String region
   String accountName
+  GoogleHttpLoadBalancer googleHttpLoadBalancer
```

Comment:
I actually went with the first option. Copying the object structure is easier than refactoring the description for the L4 operation.